### ISSUE TITLE
BB10 Screens not going back or forward

### ISSUE THREAD

**Issue by Bjern** [blackberry/bbUI.js#320]
I know this is similar to my previous issue which I said was resolved, but we're now talking about a different device.

The previous issue was about the scenario where a user goes from screen 1 -> 2 -> 3 and then tries to get back.  After going back 3 -> 2 -> 1 and clicking on a menu; screen 1 would stack screen 2 below it.  THIS IS RESOLVED on my curve phone.

However it worked fine on BB10 before I updated.  I am now using bbui 0.9.4 (yes I know its beta) and it seems to be worse.  Why?  I can go 1 -> 2 -> 3 then back 1 screen ONLY.  ie: 1 -> 2 -> 3-> 2 then it stops working.  Can't go forward or back, nothing.  Similarly I can go 1 -> 2 -> 1 then nothing.  Can always click back only once.

Is anyone else having this trouble on dev alpha with bbui?  Is there another issue being worked on I'm not aware of?  BTW I get no errors in the web inspector.  The only thing there is the info / warning message: event.layerX and event.layerY are broken and deprecated in WebKit. They will be removed from the engine in the near future.  WHICH increments every time I click on an image list.

I can supply project code, its a free app only valuable to a small niche of people. BTW how do I label this BB10 and Playbook?
